uefi-pi/src/fv.rs

```diff
@@ -19,15 +19,31 @@ use scroll::Pread;
 
 use crate::pi::fv::*;
 
+// Read FV header from slice and validate its integrity
+pub fn read_fv_header(fv_data: &[u8]) -> Option<FirmwareVolumeHeader> {
```

Comment:
Could you add some unit tests for this function?

Reply:
Sure